<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task1QED</class>
 <widget class="QDialog" name="Task1QED">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;- merge</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_a">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>83</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_b">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>236</y>
     <width>83</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Max:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>262</y>
     <width>50</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Min:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>239</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;null&gt;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>260</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;null&gt;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_a">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>83</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_a_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>266</y>
     <width>83</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Task1QED</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>10</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Task1QED</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
